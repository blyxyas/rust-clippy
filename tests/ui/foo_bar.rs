#![warn(clippy::foo_bar)]

fn main() {
    // test code goes here
    compile_error!("hi");
    //~^ error: hi
}
