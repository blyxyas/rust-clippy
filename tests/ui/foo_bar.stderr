error: hi
  --> tests/ui/foo_bar.rs:5:5
   |
LL |     compile_error!("hi");
   |     ^^^^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

